web/src/engine/websites/MangaKawaii.ts

```diff
@@ -45,23 +40,24 @@ export default class extends DecoratableMangaScraper {
     }
 
     public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
-        const englishChapters = await this.getChaptersFromLanguage(manga, 'en');
-        const frenchChapters = await this.getChaptersFromLanguage(manga, 'fr');
-        return [...englishChapters, ...frenchChapters];
+        const languages = ['eng', 'fr'];
+        const chapters = [];
+        languages.forEach(async lang => chapters.push(await this.getChaptersFromLanguage(manga,lang)));
```

Comment:
Will `.forEach` wait if the inner callback function is `async`?
May better use an ordinary `for ... of ...` loop instead just to be sure

Reply:
Just testet, this will not work, line 46 (return) is hit before the async forEach completes

Reply:
Oops